lib/rubocop/cop/method_complexity_patch.rb

```diff
@@ -0,0 +1,10 @@
+# Monkey patching! Here be dragons.
+module RuboCop
+  module Cop
+    module MethodComplexity
+      def add_offense(node, loc, message = nil, severity = nil)
```

Comment:
Yeah, thanks, I agree. Want to keep it shorter than the commit message explanation, and not sure how I want to word it yet. I'll ping you when I have something.